site/src/Librecores/ProjectRepoBundle/Validator/Constraints/UserOrgNameValidator.php

```diff
@@ -37,7 +37,8 @@ class UserOrgNameValidator extends ConstraintValidator
      *
      * @var string[]
      */
-    const RESERVED_NAMES = [ 'org', 'orgs', 'project', 'projects', 'search', 'unassigned' ];
+    const RESERVED_NAMES = [ 'org', 'orgs', 'planet', 'project', 'projects',
+                             'search', 'static', 'unassigned', 'user' ];
 
```

Comment:
Since this lookup is not in any hot path, can we, just to be safe and in addition to the list of reserved names, do a `doesThisRouteExist($userOrgName)` lookup (however that is actually called in Symfony)?